app/models/teacher_set.rb

```diff
@@ -59,8 +59,8 @@ def pending_holds_for_user(user)
     end
   end
 
-  def teacher_set_holds_count
-     holds.collect{|i| i.quantity}.inject(:+)
+  def teacher_set_holds_count(user)
```

Comment:
As long as we're changing this method .. now that this is scoped to a user, would a better name be `holds_count_for_user`? (The "teacher_set_" prefix is redundant given that it's a member function on a teacher set.)